shell/src/main/java/alluxio/cli/bundler/command/CollectLogCommand.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.bundler.command;
+
+import alluxio.client.file.FileSystemContext;
+import alluxio.conf.PropertyKey;
+import alluxio.exception.AlluxioException;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.io.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nullable;
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Command to collect Alluxio logs.
+ * */
+public class CollectLogCommand  extends AbstractInfoCollectorCommand {
```

Comment:
Can we reuse the common code from `CollectConfigCommand`?

Reply:
That's a great idea but i'm a little not sure. For a few reasons these CollectConfigCommand and CollectLogCommand may diverge more in the implementation in the future:
1. Copying config files should be made more security-aware.
2. Copying logs may require more optimization and tuning as there can be a lot. (eg. 10MB for each log and 100 log files rotation)

Reply:
In that case then we can leave it as is.